ra/registration-authority.go

```diff
@@ -40,14 +41,16 @@ type RegistrationAuthorityImpl struct {
 	log         *blog.AuditLogger
 	// How long before a newly created authorization expires.
 	authorizationLifetime time.Duration
+	rlPolicies            map[string]cmd.RateLimitPolicy
 }
 
 // NewRegistrationAuthorityImpl constructs a new RA object.
-func NewRegistrationAuthorityImpl(clk clock.Clock, logger *blog.AuditLogger) RegistrationAuthorityImpl {
+func NewRegistrationAuthorityImpl(clk clock.Clock, logger *blog.AuditLogger, policies map[string]cmd.RateLimitPolicy) RegistrationAuthorityImpl {
```

Comment:
*doh* I'd been using a map for testing fun YAML stuff, I thought I'd replaced it with a stricter struct before pushing... I guess I did not.